editor/scene_tree_editor.cpp

```diff
@@ -1297,6 +1329,10 @@ SceneTreeEditor::SceneTreeEditor(bool p_label, bool p_can_rename, bool p_can_ope
 	tree->connect("button_clicked", callable_mp(this, &SceneTreeEditor::_cell_button_pressed));
 	tree->connect("nothing_selected", callable_mp(this, &SceneTreeEditor::_deselect_items));
 
+	// Need to refresh Tree's text editor's error stylebox.
+	tree->get_text_editor()->connect("focus_entered", callable_mp(this, &SceneTreeEditor::_renaming).bind("OK"));
```

Comment:
This solution currently works, but due of the... "jank" way it looks I'm probably going to make a separate function to set the "error" Stylebox on the LineEdit